### PR TITLE
Dachoka3000/#24/ Add Project Readme

### PR DESCRIPTION
**Pull request by Dachoka3000**
#24 Create README for the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
+* Github Projects
+
+## Requirements and Specifications
+![node:v0.9.0](https://img.shields.io/badge/node-v0.9.0-blue.svg)
+![npm:v0.9.0](https://img.shields.io/badge/npm-v8.15.0-blueviolet.svg)
+![ruby:2.7.4](https://img.shields.io/badge/ruby-2.7.4-yellow.svg)
+*  `node v0.9.0`
+* `npm 8.15.0`
+* `ruby 2.7.4`
+
```
Comment (otsembo) [inline README.md:42]: Before adding the Authors, ensure you speak of how to set up the development environment using `.env` files.
Example:
image

File: README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
...
+* Ruby
+* Sinatra
+* React
+* Typescript
+* Jest
+* Firebase
+* Github Actions
+* Github Projects
+
+## Requirements and Specifications
```
Comment (Albert-Byrone) [inline README.md:35]: Move the versions to the technology used section(Create one if does not exist)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,74 @@
...
+Based on the Agile Method.
+Planning and tracking of project tasks was done using `Github Projects`.
+Sample steps:
+
+> - Check for any tasks assigned to you on the project boards.
+> - In your local environment, create a new branch and work on the task assigned to you while on that branch.
+> - Push your work to Github
+> - Create a pull request and link the ticket issue
+> - Update the details and status of that issue on the project boards.
+
```
Comment (otsembo) [inline README.md:66]: Ensure your contribution guide is detailed, explaining the exact steps it takes to make the contribution.
Find below an example:
image

**Review by Albert-Byrone — CHANGES_REQUESTED** [review]
Add the Prerequisites section and a section for setting up the application with docker  e.g 
```
Prerequisites
- node v14.17.4 and above
- npm 6.14.14 and above
- React 18.2.0
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,74 @@
-# MORINGA HUB
-Welcome to Moringa Hub!!
```
Comment (otsembo) [inline README.md:2]: You can add some shields that indicate stats about the project. Include the following:

- [ ] Issues
- [ ] Number of GitHub contributors
- [ ] GitHub repo size
- [ ] License

Use https://shields.io/ as a good reference point for all these.

**Review by otsembo — CHANGES_REQUESTED** [review]
Hello @Dachoka3000  Good Job on the PR.
Kindly have a look at the comments below for a few fixes.

**Comment by Dachoka3000** [comment]
I've made the requested changes:

- [x] Add docker setup section
- [x] Add versions in the technologies used section
- [x] Add section for setting up environment
- [x] Add a more detailed contribution guide
- [x] Add shields to indicate stats about the project. (The license shield was there already)
@Albert-Byrone  @otsembo

File: README.md [hunk 5]
```diff
@@ -1,2 +1,142 @@
...
+```
+
+* Ensure you create three environment files: `.env`, `.env.production` and `.env.development`.
+**Ensure you add any necessary variables in the appropriate file**. Example:
+```
+MONGO_DB_URI='random env here'
+## DO NOT commit your .env files to version control
+```
+
+* Once your code is in the remote branch, create a pull request to the `development` branch.
```
Comment (otsembo) [inline README.md:131]: Change the branch to `main` instead of `development`

**Review by otsembo — APPROVED** [review]
Hello @Dachoka3000 Good job on the PR. make the small change.
Everything else looks okay.

**Review by Albert-Byrone — APPROVED** [review]

**Review by Albert-Byrone — APPROVED** [review]

**Review by otsembo — APPROVED** [review]

**Review by otsembo — APPROVED** [review]
Looks good to me. Great job @Dachoka3000

**Pull request merged**